packages/tracing/test/index.bundle.test.ts

```diff
@@ -1,11 +1,9 @@
 import { Integrations } from '../src/index.bundle';
-import { testOnlyIfNodeVersionAtLeast } from './testutils';
 
 describe('Integrations export', () => {
-  // TODO `Object.values` doesn't work on Node < 8
-  testOnlyIfNodeVersionAtLeast(8)('is exported correctly', () => {
-    Object.values(Integrations).forEach(integration => {
-      expect(integration.id).toStrictEqual(expect.any(String));
+  it('is exported correctly', () => {
+    Object.keys(Integrations).forEach(key => {
+      expect(Integrations[key as keyof typeof Integrations].id).toStrictEqual(expect.any(String));
```

Comment:
I just made it so that we don't need to use `testOnlyIfNodeVersionAtLeast` anymore.

Reply:
Yeah good question on the value of this test. I just copied the existing one from `index.bundle.test.ts` into `index.test.ts`, but open to alternatives.

Reply:
> I just made it so that we don't need to use testOnlyIfNodeVersionAtLeast anymore.

I figured that might be the case. My thought about that, though, is that if you leave aside the node restriction, IMHO the old test is simpler, because you don't need the keyof typeof stuff, and since we're about to drop node 6 compatibility anyway, we could just remove the wrapper when we do and still get to have a simpler test. What do you think?

That said, yeah, first we have to figure out if we even want the test in the first place! My issue with it is less "is there value in testing the exportation?" (though I'm iffy on that) and more "okay, but how does any of this prove that it's exported correctly?". Do you understand the logic there?

Reply:
> whether to deal with it here or come back to it

I will come back to this and address this in another PR. You make great points that we have to figure out.